src/Core/System/CustomEntity/Schema/DynamicFieldFactory.php

```diff
@@ -114,22 +114,24 @@ public static function create(ContainerInterface $container, string $entityName,
         return $collection;
     }
 
-    private static function kebabCaseToCamelCase(string $string): string
+    public static function createRepository(ContainerInterface $container, EntityDefinition $definition): EntityRepository
```

Comment:
creating an EntityRepository in a class called `DynamicFieldFactory` does not sound right. probably contradicts single responsibility pattern. is there a reason, you do it here?

Reply:
that method was already there, i can put it also in CustomEntityRegistrar